crates/uv-build-backend/src/lib.rs

```diff
@@ -318,6 +318,20 @@ pub fn build_wheel(
     debug!("Writing wheel at {}", wheel_path.user_display());
     let mut wheel_writer = ZipDirectoryWriter::new_wheel(File::create(&wheel_path)?);
 
+    // Wheel excludes
+    // TODO(konstin): The must be stronger than the source dist excludes, otherwise we can get more
+    // files in source tree -> wheel than for source tree -> source dist -> wheel.
```

Comment:
Can you say why you don't want to get more files in `source tree -> wheel` versus `source tree -> source dist -> wheel`?

Reply:
Imagine we exclude `src/a/b.py` in source dist, but not in wheel. That means in source dist -> wheel, `src/a/b.py` is part of the final packages, but we remove it in source tree -> source dist, so the source dist -> wheel step can't include it (the file doesn't exist in the `.tar.gz`), so `src/a/b.py` isn't part of the final package.